apps/channels/migrations/0022_alter_experimentchannel_table.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 5.1.5 on 2025-05-16 13:02
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ocs_channels', '0021_alter_experimentchannel_platform'),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
+                migrations.AlterModelTable(
+                    name='experimentchannel',
+                    table='channels_experimentchannel',
```

Comment:
Wouldn't the new channels app prefix its tables with "channels_"?

Reply:
no, it will use `ocs_channels` since that's the app name. So since we changed the app name and also set `db_table` on the model, Django things we've renamed the model table. But since we haven't, we need to use `SeparateDatabaseAndState` to let Django update it's internal state without actually modifying the DB.

Reply:
Oh I meant the Django Channels library. Isn't that prefixing its tables with `channels_`? So shouldn't we rename our tables as well to be prefixed with `ocs_channels_`?

Reply:
Oh right. The Django Channels app doesn't have any models. I'll take another look at renaming the tables - that might just be cleaner in the long run anyway.